packages/experimental/Divider/SPEC.md

```diff
@@ -2,4 +2,161 @@
+- If `alignContent` is equal to "center" or not set, the content appears exactly in the middle of the dividing line.
+
+## API
+
+### Slots
+
+The `Divider` uses 6 total slots:
+
+- `root` [View] The outer container of the component
+- `beforeLine` [View] The dividing line rendered before passed content. If no content is passed, this is the only slot rendered.
+- `afterLine` [View] The dividing line rendered after passed content. Not rendered if no content is passed.
+- `wrapper` [View] If content is passed, this is the outer container of the content between `beforeLine` and `afterLine` which provides the correct padding for the content.
+- `text` [TextV1] If text is passed as a child, this renders inside `wrapper`.
+- `icon` [IconV1] If the `icon` prop is set, an icon will render inside `wrapper`. If text is also passed while the `icon` prop is set, then this slot will not render.
+
+These slots can be modified using the `compose` function on `Divider`. For more information on using the compose API, please see [this page](https://github.com/microsoft/fluentui-react-native/blob/main/packages/framework/composition/README.md).
+
+### Props
+
+Below is the set of props `Divider` supports.
+
+```tsx
+export interface DividerProps {
+  /**
+   * If a text or icon is passed, this dictates where content appears in the divider: at the start, centered, or towards the end.
+   * @default 'center'
+   * Note: This prop is not supported on mobile platforms(Android & iOS).
+   */
+  alignContent?: DividerAlignment;
+  /**
+   * If no color is passed, the divider and its content are colored using different theme tokens depending on the value of this prop.
+   * @default 'default'
+   * Note: This prop is not supported on mobile platforms(Android & iOS).
+   */
+  appearance?: DividerAppearance;
+  /**
+   * Pass an icon source to render an icon as content in the divider. Mutually exclusive with passing text as a child.
+   * Note: This prop is not supported on mobile platforms(Android & iOS).
+   */
+  icon?: IconProps;
+  /**
+   * The size of the Divider inset - the margin before the start and after the end of the divider.
+   * @default 0
+   * Note : For mobile platforms, the insetSize prop is only applied to start of the component.
+   */
+  insetSize?: DividerInsetSize;
+  /**
+   * Whether the divider is rendered as a horizontal line or a vertical line.
+   * @default false
+   * Note: This prop is not supported on mobile platforms(Android & iOS).
+   */
+  vertical?: boolean;
+}
+```
+
+### Styling Tokens
+
+Tokens can be used to customize the styling of the control by using the customize function on the `Divider`. For more information on using the customize API, please see [this page](https://github.com/microsoft/fluentui-react-native/blob/main/packages/framework/composition/README.md). The `Divider` has the following tokens:
```

Comment:
The customize API doesn't exist on the compose framework, so users will have to use compose to alter tokens.

Reply:
Is this true? In the test app, I'm somehow using `customize` to change the color tokens on my Divider. 
``` 
const RedDivider = Divider.customize({ contentColor: 'red', lineColor: 'red' });
```

Reply:
Whoops you're 100% correct, compressible doesn't have compose, but it does have customize. Sorry for the confusion.